plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -82,47 +35,65 @@ class AutoQueueConf:
     # name of the job
     name: str = "${hydra.job.name}"
 
-    # following parameters are SLURM specific
 
+@dataclass
+class SlurmParams(BaseParams):
+    """Slurm configuration overrides and specific parameters
+    """
```

Comment:
for slurm, some names will have changed now: `ntasks_per_node` -> `tasks_per_node` etc so that both slurm and local use the same terminology (the one used in AutoExecutor)